### DISCUSSION TITLE
lateral acceleration vs longitudinal acceleration

### DISCUSSION THREAD

**Discussion by decareano** [theOehrly/Fast-F1#410] [General]
Hi friends, as usual awesome work on the API. I am wondering if I can tap into Lat acceleration (my understanding: cornering with g forces) and Longitudinal acceleration (acceleration plus braking) in the API? Do they provide a better picture than say, throttle and braking? thanks,

**Comment by theOehrly** [comment]
Acceleration data is not available through the API. You can calculate a rough estimate from the position data but this is really not easy to do because the data is inaccurate and has a low sample rate.

**Comment by decareano** [comment]
Thank you. I understand what you mean. When I look at a scatter plot of say, lat acceleration vs speed, it looks more of an indication or a tendency to me rather than a cold hard fact. Any good reading you suggest in the matter.

**Reply by TracingInsights** [reply]
Here's how I calculate Lat and Long accelerations from telemetry data- https://gist.github.com/TracingInsights/6773604fe91f2e2b29985474088eda8f

You can also get ready-made Lat Acc vs Speed, Long Acc vs Speed and Lat Acc vs Long Acc plots from https://tracinginsights.com/analysis

**Reply by decareano** [reply]
Thanks TracingInsights. I think I will use your ready-made plots for now.
